src/Common/filesystemHelpers.cpp

```diff
@@ -87,7 +87,10 @@ BlockDeviceType getBlockDeviceType([[maybe_unused]] const String & device_id)
 #if defined(OS_LINUX)
     try
     {
-        ReadBufferFromFile in("/sys/dev/block/" + device_id + "/queue/rotational");
+        const String path = "/sys/dev/block/" + device_id + "/queue/rotational";
```

Comment:
Let's also use std::filesystem::path::operator/

Reply:
Makes sense to me. Changed.